changelog/12208.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+identity: handle creation of role without a key parameter
```

Comment:
```suggestion
identity: disallow creation of role without a key parameter
```
Is that correct? I read "handle" as "support".

Reply:
I'm also thinking this should be more of a "changes" category. Folks could definitely be using the system like this today:

1. Make a role (no key)
2. Make a key (listing allowed_client_ids that needed to come from from step 1, so very reasonable to do this second)
3. Update the role with the key

This flow works today but will break with this change. We've said in our docs all along that the key was required, so we're really just meeting that contract so I think the change is OK, but it is an API break.

Reply:
> Is that correct? I read "handle" as "support".

Yes, you are correct. I forgot to update the changelog after changing the implementation.